### PR TITLE
docs: clarify command for new repo, port-forward flags

### PR DESCRIPTION
**Pull request by joshgav**
The instructions for telling the CLI how to automatically use port-forward were intertwined with the instructions for registering a repo; this separates them.

This also moves the port-forward flags instructions to the first place in the doc a user would need them.

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [x] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [x] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [x] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [x] Does this PR require documentation updates?
* [x] I've updated documentation as required by this PR.
* [x] Optional. My organization is added to USERS.md.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [x] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [x] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -131,10 +134,11 @@ An example repository containing a guestbook application is available at
 
 ### Creating Apps Via CLI
 
-!!! note
-    You can access Argo CD using port forwarding: add `--port-forward-namespace argocd` flag to every CLI command or set `ARGOCD_OPTS` environment variable: `export ARGOCD_OPTS='--port-forward-namespace argocd'`:
+Register the guestbook repository with the ArgoCD controller with the following command:
```
Comment (jannfis) [inline docs/getting_started.md:137]: I think this should be rephrased to something like:

> Create the example guestbook application with the following command:

**Review by jannfis — CHANGES_REQUESTED** [review]
Thanks for improving the docs, @joshgav. 

I have a minor request for change, please see below.

File: docs/getting_started.md (hunk 1 above)
Comment (joshgav) [reply docs/getting_started.md]: Thanks, updated!

**Review by jannfis — APPROVED** [review]
Thanks! LGTM

**Pull request merged**